### ISSUE TITLE
Pytorch model cannot be served by ORT 0.5.0 + ONNX 1.6.0

### ISSUE THREAD

**Issue by Exlsunshine** [microsoft/onnxruntime#2132]
**Describe the bug**
Pytorch model can be converted, but the converted onnx model could not be loaded by onnxruntime. Keeps telling me:
```
File "C:\Users\x\AppData\Local\Programs\Python\Python36\lib\site-packages\onnxruntime\capi\session.py", line 29, in __init__
    self._sess.load_model(path_or_bytes)
RuntimeError: [ONNXRuntimeError] : 10 : INVALID_GRAPH : Load model from test.onnx failed:This is an invalid model. Error in Node: : Node () has input size 4 not in range [min=2, max=2].
```

**Urgency**

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Win10
- ONNX Runtime installed from (source or binary): pip
- ONNX Runtime version: 0.5.0
- Python version: 3.6
- Visual Studio version (if applicable): None
- GCC/Compiler version (if compiling from source): None
- CUDA/cuDNN version: None
- GPU model and memory: None

**To Reproduce**
Describe steps/code to reproduce the behavior:
```
import torch.onnx
from torchvision import models
import onnxruntime as rt
import os

fcn = models.segmentation.deeplabv3_resnet101(pretrained=True).eval()

path = os.path.join(os.path.dirname(__file__), "test.onnx")
input = torch.randn([1, 3, 244, 244])
torch.onnx.export(fcn, input, path, opset_version=11)

sess = rt.InferenceSession(path)
print("sess is ok")
```
**Expected behavior**
the code should print: sess is ok

**Screenshots**
```
Traceback (most recent call last):
  File "ort.py", line 12, in <module>
    sess = rt.InferenceSession(path)
  File "C:\Users\x\AppData\Local\Programs\Python\Python36\lib\site-packages\onnxruntime\capi\session.py", line 29, in __init__
    self._sess.load_model(path_or_bytes)
RuntimeError: [ONNXRuntimeError] : 10 : INVALID_GRAPH : Load model from test.onnx failed:This is an invalid model. Error in Node: : Node () has input size 4 not in range [min=2, max=2].
```
**Additional context**
Add any other context about the problem here. If the issue is about a particular model, please share the model details as well to facilitate debugging.

**Comment by dashesy** [comment]
This is opset11, you should try ORT from master or wait for the release, or export with opset10 for now. A better error message would have been great.

**Comment by Exlsunshine** [comment]
> This is opset11, you should try ORT from master or wait for the release, or export with opset10 for now. A better error message would have been great.

Hi @dashesy , thanks for the tips, opset10 does work for me, but opset10 leads to low precision, the onnx output is barely the same with original output. Maybe I should pip install latest ORT later.

**Comment by Exlsunshine** [comment]
> Please try https://test.pypi.org/project/ort-nightly

Hey Faith, thanks for the info, I'd like to try it, at the meantime, do we have any rough EAT about releasing those features to ```pip install```?

**Comment by askhade** [comment]
@Exlsunshine : onnxruntime release 1.0 is now available with ONNX 1.6 version support (supporting opset 11).

**Comment by faxu** [comment]
@Exlsunshine have you been able to verify it works with the released 1.0 version?

**Comment by Exlsunshine** [comment]
> @Exlsunshine have you been able to verify it works with the released 1.0 version?

Hi @faxu & @askhade, 

I have tried the same code with onnxruntime==1.0.0, but it doesn't work either, I got the following error:

```
Traceback (most recent call last):
  File "a.py", line 12, in <module>
    sess = rt.InferenceSession(path)
  File "D:\Program\anaconda3.201907\envs\test_new_ort\lib\site-packages\onnxruntime\capi\session.py", line 23, in __init__
    self._load_model()
  File "D:\Program\anaconda3.201907\envs\test_new_ort\lib\site-packages\onnxruntime\capi\session.py", line 35, in _load_model
    self._sess.load_model(self._path_or_bytes, providers)
onnxruntime.capi.onnxruntime_pybind11_state.Fail: [ONNXRuntimeError] : 1 : FAIL : Node: Output:1082 [ShapeInferenceError] Can't merge shape info. Both source and target dimension have values but they differ. Source=1 Target=21 Dimension=1
```
Env:
```
onnx                      1.6.0
onnxruntime               1.0.0
torch                     1.3.0+cpu
torchvision               0.4.1+cpu
```

but I can convert the same model by specifying opset_version = 10.

**Comment by askhade** [comment]
@Exlsunshine : Can you please share the model?

**Comment by Exlsunshine** [comment]
> @Exlsunshine : Can you please share the model?

It's a built-in model in torchvision, you can reproduce the error by executing(try opset_version 11 and 10, 10 works but 11 not):
```
import torch.onnx
from torchvision import models
import onnxruntime as rt
import os

fcn = models.segmentation.deeplabv3_resnet101(pretrained=True).eval()

path = os.path.join(os.path.dirname(__file__), "test.onnx")
input = torch.randn([1, 3, 244, 244])
torch.onnx.export(fcn, input, path, opset_version=11)

sess = rt.InferenceSession(path)
print("sess is ok")
```

**Comment by faxu** [comment]
@Exlsunshine sorry for the delay; checking back on this...is the issue still reproducible with the latest 1.1 ORT release?

**Comment by ysujiang** [comment]
> > @Exlsunshine : Can you please share the model?
> 
> It's a built-in model in torchvision, you can reproduce the error by executing(try opset_version 11 and 10, 10 works but 11 not):
> 
> ```
> import torch.onnx
> from torchvision import models
> import onnxruntime as rt
> import os
> 
> fcn = models.segmentation.deeplabv3_resnet101(pretrained=True).eval()
> 
> path = os.path.join(os.path.dirname(__file__), "test.onnx")
> input = torch.randn([1, 3, 244, 244])
> torch.onnx.export(fcn, input, path, opset_version=11)
> 
> sess = rt.InferenceSession(path)
> print("sess is ok")
> ```



> > @Exlsunshine have you been able to verify it works with the released 1.0 version?
> 
> Hi @faxu & @askhade,
> 
> I have tried the same code with onnxruntime==1.0.0, but it doesn't work either, I got the following error:
> 
> ```
> Traceback (most recent call last):
>   File "a.py", line 12, in <module>
>     sess = rt.InferenceSession(path)
>   File "D:\Program\anaconda3.201907\envs\test_new_ort\lib\site-packages\onnxruntime\capi\session.py", line 23, in __init__
>     self._load_model()
>   File "D:\Program\anaconda3.201907\envs\test_new_ort\lib\site-packages\onnxruntime\capi\session.py", line 35, in _load_model
>     self._sess.load_model(self._path_or_bytes, providers)
> onnxruntime.capi.onnxruntime_pybind11_state.Fail: [ONNXRuntimeError] : 1 : FAIL : Node: Output:1082 [ShapeInferenceError] Can't merge shape info. Both source and target dimension have values but they differ. Source=1 Target=21 Dimension=1
> ```
> 
> Env:
> 
> ```
> onnx                      1.6.0
> onnxruntime               1.0.0
> torch                     1.3.0+cpu
> torchvision               0.4.1+cpu
> ```
> 
> but I can convert the same model by specifying opset_version = 10.

I meet the same question, and how did you solve this problem?
